test/components/views/right_panel/__snapshots__/RoomSummaryCard-test.tsx.snap

```diff
@@ -123,10 +123,10 @@ exports[`<RoomSummaryCard /> has button to edit topic when expanded 1`] = `
         data-orientation="horizontal"
         role="separator"
       />
-      <label
```

Comment:
From https://github.com/element-hq/compound-web/pull/178 in new compound-web

Reply:
https://github.com/matrix-org/matrix-react-sdk/pull/12586 handles the same for those curious